cmd/krel/BUILD.bazel

```diff
@@ -0,0 +1,32 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_binary", "go_library")
```

Comment:
I’m wondering if we need bazel at all if we stick to go modules and a recent version (>1.12).

Reply:
@saschagrunert -- Though I grumble on occasion about bazel, we get the advantage of leveraging the verify tests from k/repo-infra.

Updating & verifying all things is now simple:
- update: `./hack/update-all.sh`
- verify: `./hack/verify-all.sh`

See here: https://github.com/kubernetes/release/pull/889, https://github.com/kubernetes/release/pull/890